packages/rulesets/src/spectral/functions/post-response-codes.ts

```diff
@@ -0,0 +1,81 @@
+// Synchronous POST must have 200 & 204 return codes and LRO POST must have 200 & 202 return codes.
+// RPC Code: RPC-Asyn-V1-11
+
+const SYNC_POST_RESPONSES = ["200", "204", "default"]
+const LR_POST_RESPONSES = ["200", "202", "default"]
```

Comment:
this will need to be modified to reflect the expectations mentioned in the comment on the md file